src/server.ts

```diff
@@ -1,3 +1,4 @@
+import * as dotenv from 'dotenv';
```

Comment:
What stuff are we using from an environment variable?

Reply:
I have a .env file in the src/ directory, it contains a variable call GOOGLE_CLIENT_ID that I use when verifying the Google auth tokens.

It gets used in src\middleware\google-auth.ts